data/transition-sites/ea_shoreham-adur-tidal-walls.yml

```diff
@@ -0,0 +1,7 @@
+---
+site: ea_shoreham_adur_tidal_walls
```

Comment:
Based on other config, this value should follow the same convention as the filename, i.e. `ea_shoreham-adur-tidal-walls`